support/schemas/user_update.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "user_create",
```

Comment:
I believe this should be `user_update` since there is another schema with the same ID